### DISCUSSION TITLE
can't unistall logisim evolution

### DISCUSSION THREAD

**Discussion by pityka3000** [logisim-evolution/logisim-evolution#2113] [Q&A] [answered]
i use windows 10, logisim evolution 3.2.7. i delete all files from folders, clear by %temp% & prefetch, howevwr still cannot delete it windows setings. an arror occur "The path cannot be found. Verify that you have access to this location. But i try this methon on other device and it works.
And 1 more problem. All zip files i want to download become circ files. That's the cause i want to unistall logisim

**Comment by davidhutchens** [comment, marked as answer]
The issue with downloading zip files was fixed in Logisim-Evolution by PR #1765. The latest release, 3.9.0 contains this fix. Installing this release may fix your problem. You might also find other suggestions by looking at that PR and at issue #1471.

As I am not a Windows user, I cannot help with Windows settings problems.
